app/javascript/tapp/appState.js

```diff
@@ -966,8 +1138,26 @@ class AppState {
       return this.get('selectedSession');
     }
 
-    getInstructorsList() {
-        return this.get('instructors.list');
+    getInstructorsList(fullData = false) {
+        if(fullData)
```

Comment:
I'm not sure if this is the right place, but the modal that shows the list of utorids for the instructor should be sorted alphabetically.  It would also be good if it included the instructor name.